app/assets/javascripts/leaflet.map.js

```diff
@@ -31,6 +32,13 @@ L.OSM.Map = L.Map.extend({
       name: I18n.t("javascripts.map.base.standard")
     }));
 
+    this.baseLayers.push(new L.OSM.CyclOSM({
+      attribution: copyright + ". " + cyclosm + ". " + terms,
+      code: "Y",
+      keyid: "opnvkarte",
```

Comment:
Typo (cyclosm)

Reply:
@Phyks I do not spot what you want to be changed.

Reply:
I guess the keyid value (`openvkarte`) is coming from a bad copy paste and should be updated. Isn't it?

Reply:
Right, fixed, thank.